### PR TITLE
Updated structure.kinds to always return tuples.

### LINKED ISSUE #66: Make `kinds` always return a Tuple.
If `kinds` is not set it should return a tuple of `None` values instead of `None` to simplify the code.

### PR DESCRIPTION
**Pull request by hdsassnick**
Closes #66

### CODE REVIEW

**Comment by Scemp** [comment]
Looks good to me! Thanks @hdsassnick for organizing.

**Review by Scemp — APPROVED** [review]
Looks good to me! Thanks @hdsassnick.

File: aim2dat/ext_interfaces/ase_atoms.py [hunk 1]
```diff
@@ -37,15 +33,13 @@ def _extract_structure_from_atoms(atoms):
...
-                tags.append(int(tag[0]))
-            else:
-                tags.append(0)
+    tags = []
+    for k in structure.kinds:
+        tag = None if k is None else re.findall(r"\d+", k)
+        if tag:
+            tags.append(int(tag[0]))
+        else:
+            tags.append(0)
```
Comment (t-reents) [inline aim2dat/ext_interfaces/ase_atoms.py:42]:
```suggestion
    for k in structure.kinds:
        tag = None if k is None else re.findall(r"\d+", k)
        tag = int(tag[0]) if tag else 0
        tags.append(tag)
```

**Review by t-reents — APPROVED** [review]
Just a small suggestion. In any case, good to go! Thanks for taking care of this change!

**Pull request merged**